packages/viem-account-ledger/src/ledger-to-account.ts

```diff
@@ -0,0 +1,66 @@
+import { CELO_DERIVATION_PATH_BASE, trimLeading0x } from '@celo/base'
+import { ensureLeading0x } from '@celo/utils/lib/address.js'
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
+import { hashMessage, serializeSignature } from 'viem'
+import { LocalAccount, toAccount } from 'viem/accounts'
+import { CeloTransactionSerializable, serializeTransaction } from 'viem/celo'
+
+import { checkForKnownToken, generateLedger } from './utils.js'
+
+type LedgerAccount = LocalAccount<'ledger'>
+
+export const CELO_BASE_DERIVATION_PATH = `${CELO_DERIVATION_PATH_BASE.slice(2)}/0`
```

Comment:
this is exported here but not from the module. is that right?

Reply:
Nope indeed